src/main/java/com/hillel/items_exchange/dao/UserRepository.java

```diff
@@ -1,13 +1,14 @@
 package com.hillel.items_exchange.dao;
 
 import com.hillel.items_exchange.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 
 @Repository
-public interface UserRepository extends CrudRepository<User, Long> {
+public interface UserRepository extends CrudRepository<User, Long>, JpaRepository<User, Long> {
```

Comment:
Interface `JpaRepository` extends `PagingAndSortingRepository` which in turn extends `CrudRepository`. So I think, you can leave only `JpaRepository<User, Long>`

Reply:
OK, thanks!

Reply:
If you delete `CrudRepository` it will not ruin this code =)